kubernetes/uptime-kuma/deployment.yml

```diff
@@ -0,0 +1,34 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  labels:
+    component: uptime-kuma
+  name: uptime-kuma
+spec:
+  selector:
+    matchLabels:
+      component: uptime-kuma
+  replicas: 3
+  strategy:
+    rollingUpdate:
+      maxSurge: 1
+      maxUnavailable: 1
+    type: RollingUpdate
```

Comment:
Given update-kuma is not stateless, and uses sqlite IIRC, I'm not sure a `RollingUpdate` is safe? I deployed with `Recreate`.